appserver/admingui/core/pom.xml

```diff
@@ -179,35 +101,5 @@
             <version>${project.version}</version>
             <scope>provided</scope>
         </dependency>
-        <dependency>
```

Comment:
TO BE FIXED: This dataprovider is repackage of the woodstock provider, but both were on classpath here. The "com.sun" comes from jsftemplating dependencies. Options:
* exclude from dependencies
* move also jsftemplating only where it is used (and other "mandatory default dependencies" too)